contracts/governance/GovernorAlpha.sol

```diff
@@ -13,25 +13,17 @@
 pragma solidity ^0.6.10;
 pragma experimental ABIEncoderV2;
 
-import {ClaimableContract} from "../trusttoken/common/ClaimableContract.sol";
+import {UpgradeableClaimable as Claimable} from "../common/UpgradeableClaimable.sol";
```

Comment:
Could you remove the "as Claimable" and just use UpgradeableClaimable throughout? We should reduce name conflicts where possible, especially if they could be confused with something else with an established meaning.

Reply:
Also it would be problematic to flatten it